test/check-terminal

```diff
@@ -26,6 +26,18 @@ class TestTerminal(MachineCase):
         m = self.machine
         b.default_user = "admin"
 
+        # XXX - figure out why locale isn't set up properly by default
+        if "debian" in m.image:
+            m.execute("echo 'en_US.UTF-8' >>/etc/locale.gen; locale-gen")
+            m.execute("echo 'export LANG=en_US.UTF-8' >>/home/admin/.bashrc")
```

Comment:
Why do we need this? Does this mean the terminal will be broken on debian unless the user performs these actions?